.gitattributes

```diff
@@ -7,3 +7,7 @@
 # Have Github ignore js vendored files.
 # https://github.com/gramps-project/gramps/tree/master/data/javascript
 #data/javascript/*.js linguist-vendored
+
+# don't mess with line endings for Gedcom files
+*.ged binary

```

Comment:
Yes, I am referring specifically to telling git to [_not_ assume it's text](https://git-scm.com/docs/gitattributes#__code_text_code). It's not the same as [telling git it's binary](https://git-scm.com/docs/gitattributes#_using_macro_attributes).